### ISSUE TITLE
Missed requirement: docker binary not found in PATH.

### ISSUE THREAD

**Issue by Merith-TK** [lucor/fyne-cross#19]
```
[developer@archlinux Workspace]$ fyne-cross --targets=windows/amd64 fyne.test.go 
Missed requirement: docker binary not found in PATH
[developer@archlinux Workspace]$ which docker
/usr/bin/docker
```

This is on my archlinux development server, the docker binary is installed via `yay -S docker`, if there is a specific type of docker you need, please make a note of it in the README

**Comment by lucor** [comment]
@Merith-TK no special type of docker is needed. 
Probably it is related to the check logic about the presence of the docker binary.
Could you please report the output of `docker version` ?

**Comment by Merith-TK** [comment]
```
[developer@archlinux Workspace]$ docker version
Client:
 Version:           19.03.4-ce
 API version:       1.40
 Go version:        go1.13.1
 Git commit:        9013bf583a
 Built:             Sat Oct 19 04:40:07 2019
 OS/Arch:           linux/amd64
 Experimental:      false
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```

**Comment by lucor** [comment]
Thanks, the error message reported by fyne-cross is misleading. The code needs to be updated to report a clear one.

The check runs `docker version` but in your case the exit code is `1` since docker daemon is not running or not available for your user (i.e. require a privileged user or `sudo`)

If docker is running, running the command with `sudo` like below should fix:

`sudo fyne-cross --targets=windows/amd64 fyne.test.go`

If you don’t want to preface the fyne-cross command with `sudo` these steps should help: https://docs.docker.com/install/linux/linux-postinstall/

Please let me know if it fixes.

**Comment by lucor** [comment]
Released v1.2.2 that should report the correct error message.
Please feel free to reopen this issue if the new version does not fixes.
